src/grein_loader/load_dataset.py

```diff
@@ -19,9 +19,9 @@
 LOGGER = logging.getLogger(__name__)
 
 
-def load_dataset(gse_id: str) -> Tuple[dict, dict, pandas.DataFrame]:
+def load_dataset(gse_id: str, download_type: str="RAW") -> Tuple[dict, dict, pandas.DataFrame]:
     """ Loads a dataset from GREIN.
-        :param: gse_id: The dataset's GSE id.
+        :param: gse_id: The dataset's GSE id, download_type: The type of data to download for expression value, either RAW or NORMALIZED
         :type: gse_id: str
         :return: description, metadata, count_matrix of the GREIN dataset
         :rtype: description:dict, metadata:dictionary, count_matrix:pandas dataframe
```

Comment:
I suggest to add a quick check to the beginning of the function to ensure that the `download_type` is valid.

ie.

```python
if download_type != "RAW" and download_type != "NORMALIZED":
   raise ValueError("Invalid download_type passed. Value must either by 'RAW' or 'NORMALIZED'.")
```